app/lib/backend.dart

```diff
@@ -640,11 +644,22 @@ Future<models.PackageVersion> parseAndValidateUpload(
     ..changelogContent = changelogContent
     ..libraries = libraries
     ..downloads = 0
```

Comment:
unrelated comment: at some point we should do some cleanup and get rid of the 'downloads' field.